.travis.yml

```diff
@@ -13,6 +13,7 @@ rvm:
   - '2.6'
   - '2.7'
   - '3.0'
+  - truffleruby-head
```

Comment:
If you could change this to:

```yaml
- 'truffleruby'
```

So that it matches style for other versions above. Also, do you need to use `head`? You normal release be fine?

Reply:
Hi @gogainda,

Not sure if you see this question here? :)

Krzysztof